cmd/agent/app/integrations.go

```diff
@@ -162,6 +112,45 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func getIntegrationName(packageName string) string {
+	switch packageName {
+	case "datadog-checks-base":
+		return "base"
+	case "datadog-checks-downloader":
+		return "downloader"
+	case "datadog-go-metro":
+		return "go-metro"
+	default:
+		return strings.Replace(strings.TrimPrefix(packageName, "datadog-"), "-", "_", -1)
+	}
+}
+
+func normalizePackageName(packageName string) string {
+	return strings.Replace(packageName, "_", "-", -1)
+}
+
+func semverToPEP440(version *semver.Version) string {
```

Comment:
Is there a reason for using semver? Why not PEP440 directly? I understand it simplifies our parsing code, but it does make things confusing for us / Support when communicating versions to users.

Reply:
Well that's the point, users should use the semver version all the way, and not care about the pep440 version.

Reply:
Okay, I'm just saying we need to be mindful of how we communicate our PEP 440 versions to Support or Users